packages/airnode-node/src/evm/fulfillments/api-calls.ts

```diff
@@ -168,7 +168,7 @@ async function submitFail(
       request.airnodeAddress,
       request.fulfillAddress,
       request.fulfillFunctionId,
-      errorMessage,
+      errorMessage.length > 100 ? errorMessage.substring(0, 97).concat('...') : errorMessage,
```

Comment:
Can we maybe move this constant here: https://github.com/api3dao/airnode/blob/an459-string32-bool-to-abi/packages/airnode-node/src/constants.ts ?

Reply:
Done- thanks for the suggestion